packages/client/builtin/Mermaid.vue

```diff
@@ -25,7 +26,7 @@ const props = defineProps<{
 
 const vm = getCurrentInstance()
 const el = ref<HTMLDivElement>()
-const svgObj = computed(() => renderMermaid(props.code || '', Object.assign({ theme: props.theme }, vm!.attrs)))
+const svgObj = computed(() => renderMermaid(props.code || '', Object.assign({ theme: (isDark.value ? 'dark' : props.theme) }, vm!.attrs)))
```

Comment:
The user-specified theme should take the highest priority. We should change the theme as default.